controls/sshd_spec.rb

```diff
@@ -38,6 +38,9 @@
                               ssh_crypto.valid_privseparation
                             end
 
+sshd_valid_hostkeys = ssh_crypto.valid_algorithms.map { |alg| "#{sshd_custom_path}/ssh_host_#{alg}_key" }
+sshd_valid_hostkeys = sshd_valid_hostkeys[0] if sshd_valid_hostkeys.length == 1
```

Comment:
I think that code should be part of the control similar to:

https://github.com/dev-sec/linux-baseline/blob/master/controls/sysctl_spec.rb#L388-L411

Reply:
It is normally solved in the last commit.